lib/index.js

```diff
@@ -38,11 +40,15 @@ function initialize(config) {
   });
 }
 
-function getAllFileObjects() {
+function getAllFileObjects(config) {
   console.log("Fetching schema...");
   return schemas.get().then(function(res){
     console.log("Fetching all objects with files...");
-    var schemasWithFiles = onlyFiles(res);
+    if (config.onlyFiles){
```

Comment:
could we do this feature 'additive' instead of file or array? Like just files, or files + arrays?